lib/bom.js

```diff
@@ -714,7 +717,30 @@ declare class AudioContext {
   decodeAudioData(arrayBuffer: ArrayBuffer, decodeSuccessCallback: Function, decodeErrorCallback: Function): void;
   decodeAudioData(arrayBuffer: ArrayBuffer): Promise<AudioBuffer>;
   resume(): Promise<void>;
+}
+
+declare class AudioTimestamp {
+  contextTime: number;
+  performanceTime: number;
+}
```

Comment:
`AudioTImestamp` is defined here:
https://webaudio.github.io/web-audio-api/#dictdef-audiotimestamp